src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@whipermr5 here is how it looks with the currently committed changes:

**Success modal:**

<img width="667" alt="modal 2 padding" src="https://user-images.githubusercontent.com/18088721/28308467-e6aa7180-6bc3-11e7-9531-83053dcd5b4e.png">

**Single line status:**

<img width="818" alt="status 2 padding" src="https://user-images.githubusercontent.com/18088721/28308478-ef1774c6-6bc3-11e7-9b46-5a32ad3121ff.png">

**Multi line status:**
<img width="831" alt="multi status 2 padding" src="https://user-images.githubusercontent.com/18088721/28308488-f9e01124-6bc3-11e7-97b7-d10ea081fe0b.png">

I didnt realise while committing my previous changes that `padding-top .2em` was causing the status messages to look off. 

Using `padding-top: .15em` is the only one that makes the statuses look slightly better while maintaining the centred-ness on the modal.

**Screenshots with `padding-top: .15em`:

Modal:
<img width="662" alt="modal 15 padding" src="https://user-images.githubusercontent.com/18088721/28309025-b2f60758-6bc5-11e7-93bf-b731217f3473.png">

Single-line status message:
<img width="822" alt="staus 15" src="https://user-images.githubusercontent.com/18088721/28309069-d46d0cec-6bc5-11e7-88cc-0bfede4cb16a.png">

Multi-line status message:
<img width="823" alt="multi 15" src="https://user-images.githubusercontent.com/18088721/28309095-e33e6306-6bc5-11e7-932c-16922f15edbd.png">

There doesn't seem to be a padding-top amount that will satisfy both the modal and status messages completely. What do you suggest?